looker/definitions/ads.toml

```diff
@@ -94,3 +136,44 @@ friendly_name = "PPA Measurements"
 description = "Aggregated conversion data from Privacy-Preserving Attribution (PPA)"
 columns_as_dimensions = true
 submission_date_column = "submission_date"
+
+
+[data_sources.native_desktop_ad_metrics_hourly]
```

Comment:
I wonder if the issue we're seeing in CircleCI is occurring due to the underlying table requirement of having to query with a filter over `submission_date`...

Reply:
@sergiosonline I added a WHERE clause to the query: `where submission_date = @submission_date`